python/sklearn.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
+
+COPY . .
+RUN pip install --upgrade pip && pip install -e ./kfserving
+RUN pip install -e ./sklearnserver
+COPY sklearnserver/model.joblib /tmp/models/model.joblib
```

Comment:
Since this image will be used for all users, I think we should not package specific model here.

Reply:
yes. This is point in time for testing purposes until we get s3/gcs download implemented